src/main/java/uk/gov/hmcts/reform/judicialapi/configuration/FeignInterceptorConfiguration.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.hmcts.reform.judicialapi.configuration;
+
+import feign.RequestInterceptor;
+
+import java.util.Enumeration;
+
+import javax.servlet.http.HttpServletRequest;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+
+import static uk.gov.hmcts.reform.authorisation.filters.ServiceAuthFilter.AUTHORISATION;
+
+
+@Slf4j
+public class FeignInterceptorConfiguration {
+
+    @Value("${loggingComponentName}")
+    private String loggingComponentName;
+
+    @Autowired
+    AuthTokenGenerator authTokenGenerator;
+
+    @Bean
+    public RequestInterceptor requestInterceptor(FeignHeaderConfig config) {
+        return requestTemplate -> {
+            ServletRequestAttributes attrs = (ServletRequestAttributes) RequestContextHolder.getRequestAttributes();
+            if (attrs != null) {
```

Comment:
use nonNull it its trivial but let we should consistent across all application. Same for all hard code null

.